ruff_lsp/server.py

```diff
@@ -3,20 +3,27 @@
 from __future__ import annotations
 
 import asyncio
+import enum
 import json
 import logging
 import os
 import re
 import shutil
 import sys
 import sysconfig
+from collections.abc import Iterable, Mapping
```

Comment:
Just confirming that these are available on Python 3.7. Assuming yes given that CI passed.

Reply:
Yes, I'm developing locally on 3.7 :)